haystack/utils/app_loading.py

```diff
@@ -21,7 +21,7 @@ def haystack_load_apps():
 def haystack_get_models(label):
     try:
         app_mod = apps.get_app_config(label)
-        return app_mod.get_models()
+        return sorted(app_mod.get_models(), key=lambda model: u'%s.%s' % (model._meta.app_label, model._meta.object_name))
```

Comment:
This feels like this should perhaps have a comment — I'm assuming the intent was just stable ordering across runs? — and I was wondering whether it needs to be a string sort rather than just letting it sort on the values (`key=lambda model: (model._meta.app_label, model._meta.object_name)`).